docs/development.md

```diff
@@ -46,19 +47,91 @@ centos7-devel:
         "
 ```
 
+Next, you may want to edit the Vagrantfile to forward some ports from the guest to the host:
+
+```
+Vagrant.configure("2") do |config|
+  config.vm.network :forwarded_port, guest: 443, host: 8443, host_ip: "0.0.0.0"
+  config.vm.network :forwarded_port, guest: 3808, host: 3808, host_ip: "0.0.0.0"
+end
+```
+
+The above assumes you want to access the web interface via the nginx reverse proxy rather than directly via TCP port 3000.
+
 Lastly, spin up the box:
 
 ```
 vagrant up centos7-devel
 ```
 
-The box can now be accessed via ssh and the Rails server started directly (this assumes you are connecting as the default `vagrant` user):
+If you get an error about the host CPU not providing the 'svm' feature but you most assuredly do have an Intel CPU and virtualization is most definitely turned on, add this to your Vagrantfile.
+
+```
+Vagrant.configure("2") do |config|
+  config.vm.provider :libvirt do |domain|
+    domain.cpu_mode = "host-passthrough"
+  end
+end
+```
+
+Try to bring it up again with `vagrant up centos7-devel` after making that change.
+
+If you get an error related to your user not having privileges to execute libvirt commands, create a polkit rule similar to the following:
+
+```
+polkit.addRule(function(action, subject) {
+    if (action.id == "org.libvirt.unix.manage" &&
+        subject.isInGroup("wheel")) {
+            return polkit.Result.YES;
+    }
+});
+```
+
+This rule will allow any user in the `wheel` group to perform libvirt commands. The file needs to reside in `/etc/polkit-1/rules.d`, named something like `49-org.libvirt.unix.manage.rules`. Now try to bring it up again with `vagrant up centos7-devel`.
+
+You may experience some failures during the provisioning process. You may also find that rerunning `vagrant provision centos7-devel` will eventually result in a successful installation. If not, check error messages and logs for hints and/or seek help from the mailing list or IRC.
+
+Once successfulling installed, access the box via ssh: `vagrant ssh centos7-devel`.
+
+Before we can run the app, we need to install a couple things by hand to make webpack work.
+
+Install `rh-nodejs8-npm` and `http-parser-devel`:
```

Comment:
Interesting. I opened https://github.com/theforeman/puppet-katello_devel/pull/96 some time ago to automate this but wasn't aware of that SCL. I like it.